client/packages/common/src/intl/locales/en/common.json

```diff
@@ -264,6 +264,7 @@
   "label.in-packs-of": "in packs of",
   "label.in-store": "In Store (packs)",
   "label.installation-date": "Installation date",
+  "label.invoice": "Invoice",
```

Comment:
We've avoided using `Invoice` as a term in preference to `Shipment`. What's this one for? Can we change to Shipment?

Reply:
Fixed: [2580264](https://github.com/msupply-foundation/open-msupply/pull/3943/commits/2580264f2fd9a6bfa4a1b5a88db592d71c35774b)